lib/checks/color/color-contrast-evaluate.js

```diff
@@ -188,4 +148,64 @@ function colorContrastEvaluate(node, options, virtualNode) {
   return isValid;
 }
 
-export default colorContrastEvaluate;
+function findPseudoElement(vNode, {
+  pseudoSizeThreshold = 0.25,
+  ignorePseudo = false
+}) {
+  if (ignorePseudo) {
+    return;
+  }
+  const rect = vNode.boundingClientRect;
+  const minimumSize = rect.width * rect.height * pseudoSizeThreshold;
+  do {
+    const beforeSize = getPseudoElementArea(vNode.actualNode, ':before')
+    const afterSize = getPseudoElementArea(vNode.actualNode, ':after')
+    if (beforeSize + afterSize > minimumSize) {
+      return vNode // Combined area of before and after exceeds the minimum size
```

Comment:
Could you rather comment on _why_ you decided to do them both combined rather than separate checks against the minimumSize? That would be more helpful when trying to debug this later